spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerProperties.java

```diff
@@ -756,7 +749,6 @@ public EOSMode getEosMode() {
 	@SuppressWarnings("deprecation")
```

Comment:
We don't need this any more.

At the same time why do we need this `EOSMode` at all since there is only one supported mode left?..

Reply:
I left it, in case they come up with `EOSMode.V3` 😄